src/index.ts

```diff
@@ -1,5 +1,4 @@
 import { PluginFunction, Types } from "@graphql-codegen/plugin-helpers";
```

Comment:
The main change in this file is to use my new `getFileReference` function instead of `maybeImport`.